src/org/zaproxy/zap/extension/spider/OptionsSpiderPanel.java

```diff
@@ -224,7 +239,7 @@ public void initParam(Object obj) {
 		getChkParseGit().setSelected(param.isParseGit());
 		getComboHandleParameters().setSelectedItem(param.getHandleParameters());
 		getHandleODataSpecificParameters().setSelected(param.isHandleODataParametersVisited());
-
+		getIrrelevantUrlParameters().setText(param.getIrrelevantUrlParametersAsString());

```

Comment:
The undoable edits should be discarded (also after saving).

Reply:
What do you mean under undoable edits? When user type something but doesn't save it?

Reply:
Yeah, `ZapTextArea` keeps track of edits that can be undone, those should be discarded when the dialogue is closed and when initialising.